extensions/amp-video-iframe/amp-video-iframe.md

```diff
@@ -154,51 +56,18 @@ Include an `amp-video-iframe` on your AMP document:
-
-As a vendor, you can serve a generic [integration document](#integration) that references provided videos via URL parameters. AMP authors who use your video service only need to include an `<amp-video-iframe>` tag in their documents:
-
-```html
-<!--
-  data-param-* attributes are added to src and poster, so this would use the
-  following composed urls:
-
-  src: https://vendor.example/amp-video-iframe
-      ?videoid=MY_VIDEO_ID
-      &channelid=MY_CHANNEL_ID
-
-  poster: https://vendor.example/poster.jpg
-      ?videoid=MY_VIDEO_ID
-      &channelid=MY_CHANNEL_ID
--->
-<amp-video-iframe
-  layout="responsive"
-  width="16"
-  height="9"
-  src="https://vendor.example/amp-video-iframe"
-  poster="https://vendor.example/poster.jpg"
-  data-param-videoid="MY_VIDEO_ID"
-  data-param-channelid="MY_CHANNEL_ID"
->
-</amp-video-iframe>
-```
-
-The `src` and `poster` URLs are appended with [`data-param-*` attributes as query string](#data-param).
+The `src` attribute points to `my-video-player.html`, the inner document loaded inside the iframe. This document plays the video. It must include and bootstrap an [integration script](../../spec/amp-video-interface.md). The integration script allows `<amp-video-iframe>` to coordinate the video's playback.
 
-The `/amp-video-iframe` document bootstraps the [integration script](#integration) so that the AMP document can coordinate with the player.
+The `<amp-video-iframe>` component differs from [`<amp-iframe>`](../amp-iframe/amp-iframe.md) and non-AMP iframeas in the following ways:
 
-Note: If you're a vendor hosting an integration document, feel free to [contribute a code sample to this page,](https://github.com/ampproject/amphtml/blob/master/extensions/amp-video-iframe/amp-video-iframe.md) specifying your provided
-`src` and usable `data-param-*` attributes.
+- `<amp-video-iframe>` implements all [video features](../../spec/amp-video-interface.md). Features include but are not limited to autoplay, minimize-to-corner and rotate-to-fullscreen.
+- `<amp-video-iframe>` must request resources via HTTPS.
+- `<amp-video-iframe>` is not scrollable.
+- `<amp-video-iframe>` enforces sandboxing to itself and child iframes. This discourages misuse of <amp-video-iframe> as a means for advertising. 
 
-## <a id="integration"></a> Integration inside the frame
+### Video player integration
 
-In order for the video integration to work, the embedded document (e.g. `my-video-player.html`) must include a small library:
+The `<amp-video-iframe>` component requires the embedded html document to include the [video-iframe-integration library](../../src/video-iframe-integration.js). The library is separate from the component's required script. This is because the embedded html document is non-AMP and iframed.
```

Comment:
I added some clarification. But I just wanted people to know where to find it instead of just saying "import this library you know nothing about". I'm sure most people don't really care... but I can reword it again if the update is still problematic